### PR TITLE
Support multiarch/platfrom in find-release-tars

### PR DESCRIPTION
**Pull request by ruquanzhao**
Signed-off-by: Ruquan Zhao ruquan.zhao@arm.com

#### What type of PR is this?
/kind bug

#### What this PR does / why we need it:
This PR fixes `find-release-tars` to look for right path when env `KUBE_BUILD_PLATFROMS` is set.
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by ruquanzhao** [comment]
relelated issue: https://github.com/kubernetes-sigs/kubetest2/pull/240

File: cluster/common.sh [hunk 1]
```diff
@@ -328,15 +328,20 @@ function find-tar() {
 #   SERVER_BINARY_TAR
 #   KUBE_MANIFESTS_TAR
 function find-release-tars() {
-  SERVER_BINARY_TAR=$(find-tar kubernetes-server-linux-amd64.tar.gz)
+  # Use first item in KUBE_BUILD_PLATFORMS as server platform
+  KUBE_BUILD_PLATFORMS=${KUBE_BUILD_PLATFORMS:-"linux/amd64"}
+  SERVER_PLATFORM=$(cut -d' ' -f1 <<< $KUBE_BUILD_PLATFORMS)
+  OS=$(cut -d'/' -f1 <<< $SERVER_PLATFORM)
```
Comment (upodroid) [inline cluster/common.sh:334]:
```suggestion
  OS=$(cut -d'/' -f1 <<< "${SERVER_PLATFORM}")
```

File: cluster/common.sh [hunk 2]
```diff
@@ -328,15 +328,20 @@ function find-tar() {
...
+  ARCH=$(cut -d'/' -f2 <<< $SERVER_PLATFORM)
+  SERVER_BINARY_TAR=$(find-tar kubernetes-server-${OS}-${ARCH}.tar.gz)
```
Comment (upodroid) [inline cluster/common.sh:336]:
```suggestion
  SERVER_BINARY_TAR=$(find-tar kubernetes-server-"${OS}"-"${ARCH}".tar.gz)
```

**Review by upodroid — CHANGES_REQUESTED** [review]
Can you fix these lint errors please? thanks

**Pull request merged**